### ISSUE TITLE
[Bug]: FluentUI PeoplePicker: 'Load More' Button in GetMoreResults Lacks Proper Focus Indicator and Fails Luminosity Contrast

### ISSUE THREAD

**Issue by EbramTawfik** [microsoft/fluentui#32854]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz
    Memory: 26.01 GB / 47.71 GB
  Browsers:
    Edge: Chromium (120.0.2210.144)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting an Accessibility issue?

yes

### Reproduction

https://stackblitz.com/edit/vitejs-vite-6z7abl?file=src%2fPeoplePickerClassExample.tsx

### Bug Description

 The 'Load More' button within the GetMoreResults functionality of the FluentUI PeoplePicker component fails the luminosity contrast ratio for the keyboard focus indicator. Additionally, it lacks a proper border focus indicator, which poses accessibility concerns. 
https://stackblitz.com/edit/vitejs-vite-6z7abl?file=src%2fPeoplePickerClassExample.tsx
![image](https://github.com/user-attachments/assets/c7dcb2b4-882e-435b-a6dc-1ea07fced838)

### Requested priority

High

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by spmonahan** [comment]
@EbramTawfik This is fixed in the latest release, `8.120.10`.

I've [updated your example to this version](https://stackblitz.com/edit/vitejs-vite-oosk2n?file=package.json) and the focus outline is now present.

![Image](https://github.com/user-attachments/assets/8bd1f31e-679e-43ca-8296-77d6c32e6bca)
